bin/migrate.py

```diff
@@ -78,9 +80,9 @@ def open_db_get_cursor():
 def migrate_add_column(col, script):
     with open_db_get_cursor() as (cursor, conn):
         if test_column(col, cursor):
-            print 'Column (' + col + ') already present'
+            print('Column (' + col + ') already present')
```

Comment:
Make use of [f-strings](https://docs.python.org/3/whatsnew/3.6.html#whatsnew36-pep498) (which was introduced in Python 3.6) instead. It's much more cleaner.